booking-engine-api/operations/reservations.md

```diff
@@ -77,3 +77,171 @@ Get a price quotation for a specific hotel, date interval, room category and per
+            "SelectedByDefault": false,
+            "BillAsPackage": false,
+            "OfferToCustomer": true,
+            "ExcludePriceFromOffer": false,
+            "OfferToEmployee": true
+          },
+          "ChargingMode": "Once",
+          "TotalAmount": {
+            "Currency": "EUR",
+            "GrossValue": 3.0,
+            "NetValue": 3.0,
+            "TaxValues": [],
+            "Breakdown": {
+              "Items": [
+                {
+                  "TaxRateCode": null,
+                  "NetValue": 3.0,
+                  "TaxValue": 0.0
+                }
+              ]
+            }
+          }
+        }
+      ]
+    }
+  ]
+}
+```
+
+#### ReservationPrice
+
+| Property | Type | Contract | Description |
+| :-- | :-- | :-- | :-- |
+| `Identifier` | string | required | Unique identifier for reservation provided in the request data. |
+| `AmountToChargeOnConfirmation` | [Total Amount](#totalamount) | optional | Amount that needs to be charged on payment gateway on reservationGroup confirmation. |
+| `TotalAmount` | [Total Amount](#totalamount) | required | Total amount of the reservation. |
+| `ProductOrderPrices` | array of [ProductOrderPrice](#productorderprice) | required | List of product order prices assigned to the reservation. |
+
+#### ProductOrderPrice
+| Property | Type | Contract | Description |
+| :-- | :-- | :-- | :-- |
+| `ProductId` | string | required | Unique identifier of product. |
+| `AgeCategoryId` | string | optional | Identifier of age category. |
+| `ProductName` | [Localized text](hotels.md#localized-text) | required | Name of the hotel. |
+| `ProductOptions` | [ProductOptions](#productoptions) | required | Product options. |
+| `ChargingMode` | string [Product charging mode](hotels.md#product-charging-mode) | required | Charging mode of the product. |
+| `TotalAmount` | [Total Amount](#totalamount) | required | Total amount of product. |
+
+#### ProductOptions
+| Property | Type | Contract |Description |
+| :-- | :-- | :-- | :-- |
+| `SelectedByDefault` | boolean | required  | If product was selected by default for reservation. |
+| `BillAsPackage` | boolean | required  | Product is part of a package. |
+| `OfferToCustomer` | boolean | required  | Product is available in booking engine. |
+| `ExcludePriceFromOffer` | boolean | required  | Product was not available in booking engine, but it is counted in reservation total price ( eg. CityTax ). |
+| `OfferToEmployee` | boolean | required  | Product available in mews operations. |
+
+#### TotalAmount
+| Property | Type | Contract | Description |
+| :-- | :-- | :-- | :-- |
+| `Currency` | string | required | ISO 4217 code of the currency. |
+| `GrossValue` | number | required | Gross value of the amount. (Net + sum of `TaxValues`). |
+| `NetValue` | number | required | Net value of the amount. |
+| `TaxValues` | array of [Tax values](hotels.md#tax-value) | required | Tax values of the amount. **@Deprecated**. |
```

Comment:
**@Deprecated** Does this mean that the property `TaxValues` is deprecated, i.e. is still present and supported but should no longer be used? If this is the case, then (1) I would suggest using the notation used in the CHM API, which is strikethrough followed by the word 'Deprecated' (see [Channel in Process Group](https://mews-systems.gitbook.io/channel-manager-api/mews-operations/operations/reservations#process-group) for an example); (2) we should add an entry in the [Deprecations](https://mews-systems.gitbook.io/booking-engine-guide/booking-engine-api/deprecations) table.

Reply:
Nice, this is what I was looking for, will do :-)